annotations/firefox_ios/metrics/app.opened_as_default_browser/README.md

```diff
@@ -0,0 +1,3 @@
+See [here](https://sql.telemetry.mozilla.org/queries/77816/#193601) for an example of a query that uses this probe to estimate the number of new Firefox for iOS profiles that are opened as default over time. The query uses the baseline ping to calculate the number of daily new profiles, and left joins onto a count of profiles that have a non-0 (true) value for `app.opened_as_default_browser`.
```

Comment:
What do you think about using this GitHub-inspired convention to make it clear that the query is defined on STMO? I assume we can use a similar system for Looker. Also we should mark this query as requiring SSO per the contributing guidelines.

```suggestion
See [STMO#77816](https://sql.telemetry.mozilla.org/queries/77816/#193601) (SSO) for an example of a query that uses this probe to estimate the number of new Firefox for iOS profiles that are opened as default over time. The query uses the baseline ping to calculate the number of daily new profiles, and left joins onto a count of profiles that have a non-0 (true) value for `app.opened_as_default_browser`.
```